src/web/app/pages-instructor/instructor-session-edit-page/instructor-session-edit-page.component.html

```diff
@@ -39,7 +39,7 @@
       </div>
       <a type="button" class="btn btn-link" target="_blank" rel="noopener noreferrer" tmRouterLink="/web/instructor/help" [queryParams]="{questionId: 'questions', section: 'questions'}"><i class="fas fa-info-circle"></i></a>
       <button id="btn-copy-question" class="btn btn-primary" (click)="copyQuestionsFromOtherSessionsHandler()" [disabled]="isCopyingQuestion"><tm-ajax-loading *ngIf="isCopyingQuestion"></tm-ajax-loading>&nbsp;Copy Question</button>
-      <button class="btn btn-primary btn-margin-left" (click)="doneEditingHandler()">Done Editing</button>
+      <button class="btn btn-primary btn-margin-left" (click)="doneEditingHandler()">Discard unsaved edits</button>
```

Comment:
When you remove the button remember to remove the `doneEditingHandler` method in the `.ts` file as well yea